x/mongo/driver/topology/server.go

```diff
@@ -676,23 +685,29 @@ func (s *Server) check() (description.Server, error) {
 			baseOperation = baseOperation.TopologyVersion(previousDescription.TopologyVersion).
 				MaxAwaitTimeMS(maxAwaitTimeMS)
 			s.conn.setCanStream(true)
+
 			err = baseOperation.Execute(s.heartbeatCtx)
 		default:
 			// The server doesn't support the awaitable protocol. Set the socket timeout to connectTimeoutMS and
 			// execute a regular heartbeat without any additional parameters.
 
 			s.conn.setSocketTimeout(s.cfg.heartbeatTimeout)
+
 			err = baseOperation.Execute(s.heartbeatCtx)
 		}
+		duration = int64(time.Since(start))
+
 		if err == nil {
 			tempDesc := baseOperation.Result(s.address)
 			descPtr = &tempDesc
+			s.publishServerHeartbeatSucceededEvent(s.conn.ID(), duration, tempDesc, streamable)
```

Comment:
When calling `publishServerHeartbeatStartedEvent`, you use `s.conn.getCurrentlyStreaming() || streamable` as the value for the `await` field but here you just use `streamable`. Are these intentionally different?